### PR TITLE
fast_double_parser@0.8.0

### PR DESCRIPTION
**Pull request by BYVoid**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (fast_double_parser) have been updated in this PR. Please review the changes.

**Comment by BYVoid** [comment]
https://github.com/lemire/fast_double_parser/archive/refs/tags/v0.8.0.tar.gz is the only viable download url.

File: modules/fast_double_parser/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/lemire/fast_double_parser",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```
Comment (Wyverald) [inline modules/fast_double_parser/metadata.json:5]: would you mind listing yourself as the maintainer here? it should be very low effort :)

**Review by Wyverald — DISMISSED** [review]

File: modules/fast_double_parser/metadata.json (hunk 1 above)
Comment (BYVoid) [reply modules/fast_double_parser/metadata.json]: Okay.

**Review by Wyverald — APPROVED** [review]

**Pull request merged**